staging/src/k8s.io/component-base/logs/api/v1/options.go

```diff
@@ -55,9 +56,20 @@ const (
 )
 
 // NewLoggingConfiguration returns a struct holding the default logging configuration.
+// The initial verbosity is the same as currently configured in klog.
```

Comment:
The latter.

> Since this is using loggingFlags, which I think is isolated from the flagset changed by SetDefaultVerbosity, I also am not sure this would even notice if something else in process called SetDefaultVerbosity

This is a different flag set, but the same underlying flag and value in that flag, so `SetDefaultVerbosity` has an effect.

> I also don't see us actually using NewLoggingConfiguration in any non-test non-example paths in tree... should this be made more explicitly test-only?

Perhaps.

But as this really only is a problem in tests and perhaps only in one test (scheduler_perf), let's simply keep `NewLoggingConfiguration` as is and I'll look into changing the default verbosity differently in scheduler_perf. I think I took a wrong turn when trying to address this in component-base...

See https://github.com/kubernetes/kubernetes/pull/125562

/close